src/main/java/uk/gov/hmcts/reform/refunds/dtos/responses/FeeDto.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.reform.refunds.dtos.responses;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.math.BigDecimal;
+import java.util.Date;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Builder(builderMethodName = "feeDtoWith")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class FeeDto {
```

Comment:
You don;t need all these fields. Only use the ones that payment actually returns